libs/ledger-live-common/src/featureFlags/defaultFeatures.ts

```diff
@@ -583,6 +583,16 @@ export const DEFAULT_FEATURES: Features = {
   llmSolanaNfts: DEFAULT_FEATURE,
   largemoverLandingpage: DEFAULT_FEATURE,
   llmMmkvMigration: DEFAULT_FEATURE,
+  lldModularDrawer: {
+    ...DEFAULT_FEATURE,
+    params: {
+      add_account: false,
+      earn_flow: false,
+      live_app: false,
+      receive_flow: false,
+      send_flow: false,
```

Comment:
IMO it would be easier for us to have these true by default. The main `enabled` boolean will still be `false` by default anyway